tests/integration/sync/content/syncContent.test.ts

```diff
@@ -0,0 +1,106 @@
+import * as fsPromises from "fs/promises";
+
+import { expectSameSyncContentEnvironments } from "../../importExport/utils/compare";
+import { runCommand } from "../../utils/runCommand";
+import { withTestEnvironment } from "../../utils/setup";
+
+dotenvConfig();
+
+const { SYNC_SOURCE_TEST_ENVIRONMENT_ID, API_KEY, SYNC_TARGET_TEST_ENVIRONMENT_ID } = process.env;
+
+if (!API_KEY) {
+  throw new Error("API_KEY env variable was not provided.");
+}
+if (!SYNC_SOURCE_TEST_ENVIRONMENT_ID) {
+  throw new Error("SYNC_SOURCE_TEST_ENVIRONMENT_ID environment variable is not defined.");
+}
+if (!SYNC_TARGET_TEST_ENVIRONMENT_ID) {
+  throw new Error("SYNC_TARGET_TEST_ENVIRONMENT_ID env variable was not provided.");
+}
+
+describe("Sync two environments with credentials", () => {
+  it.concurrent(
+    "sync-content source environment to target environment without linked item",
+    withTestEnvironment(SYNC_TARGET_TEST_ENVIRONMENT_ID, async (environmentId) => {
+      const command =
+        `sync-content run -s=${SYNC_SOURCE_TEST_ENVIRONMENT_ID} --sk=${API_KEY} -t=${environmentId} --tk=${API_KEY} -l=default --items sync_item_1 --verbose --skipConfirmation`;
+
+      await runCommand(command);
+
+      await expectSameSyncContentEnvironments(environmentId, SYNC_SOURCE_TEST_ENVIRONMENT_ID, ["sync_item_1"]);
+    }),
+  );
+
+  it.concurrent(
+    "Sync source environment to target environment directly from source environment with linked item",
+    withTestEnvironment(SYNC_TARGET_TEST_ENVIRONMENT_ID, async (environmentId) => {
+      const command =
+        `sync-content run -s=${SYNC_SOURCE_TEST_ENVIRONMENT_ID} --sk=${API_KEY} -t=${environmentId} --tk=${API_KEY} -l=default --items sync_item_1 --depth 2 --verbose --skipConfirmation`;
+
+      await runCommand(command);
+
+      await expectSameSyncContentEnvironments(environmentId, SYNC_SOURCE_TEST_ENVIRONMENT_ID, [
+        "sync_item_1",
+        "sync_linked_item_1",
+      ]);
+    }),
+  );
+
+  it.concurrent(
+    "Sync source environment to target environment directly from source environment get item byContentType with linked item",
+    withTestEnvironment(SYNC_TARGET_TEST_ENVIRONMENT_ID, async (environmentId) => {
+      const command =
+        `sync-content run -s=${SYNC_SOURCE_TEST_ENVIRONMENT_ID} --sk=${API_KEY} -t=${environmentId} --tk=${API_KEY} -l=default --byTypesCodenames no_change_linked_type no_change_base_type --verbose --skipConfirmation`;
+
+      await runCommand(command);
+
+      await expectSameSyncContentEnvironments(environmentId, SYNC_SOURCE_TEST_ENVIRONMENT_ID, [
+        "sync_item_1",
+        "sync_draft_linked_item_1",
+        "sync_item_2",
+        "sync_linked_item_2",
+        "sync_draft_linked_item_2",
+        "sync_draft_item_1",
+        "sync_linked_item_1",
+      ]);
+    }),
+  );
+});
+
+describe("Sync environment from folder", () => {
+  const relativePath = "./tests/integration/sync/content/sourceContent.zip";
```

Comment:
The path isn't git-ignored so every run will create a file to be committed. Can you ignore it, please? Also, all other such files are in a `/data/` folder, can you put it there as well for consistency's sake?